### DISCUSSION TITLE
Am I stupid, or is this an issue?

### DISCUSSION THREAD

**Discussion by Fablor9900** [Tyrrrz/DiscordChatExporter#731] [Q&A]
I am unable to launch DCE due to lacking the following dependencies:
Windows Update KB2999226
Windows Update KB3063858
Windows Update KB3154518

Upon attempting to install them through the client, I get the following Error Message.
![image](https://user-images.githubusercontent.com/56175393/139589429-810900b4-d7d2-4e9a-958b-82fdd90db4e6.png)
I do not know if this is user error or if I need to open a bug report.

**Comment by Tyrrrz** [comment]
This is a Windows 7 issue. For now just press "Ignore".
